lib/jekyll/converters/markdown.rb

```diff
@@ -43,7 +42,7 @@ def get_processor
       # are not in safe mode.)
 
       def valid_processors
-        %w(rdiscount kramdown) + third_party_processors
+        %w(kramdown) + third_party_processors
```

Comment:
lets use `#concat` or `#unshift` here to avoid creating another Array for no reason..
```ruby
%w(kramdown).concat(third_party_processors)
```